styles/raindrop/catppuccin.user.css

```diff
@@ -0,0 +1,172 @@
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    &,
+    [data-theme] {
+      --background-color: @base;
+      --disable-background-color: @crust;
+      --disable-text-color: @overlay1;
+      --accent-color: @accent-color;
+      --alternative-background-color: @mantle;
+      --sidebar-background-color: @mantle;
+      --hover-background-color: fade(@overlay0, 20%);
+      --active-background-color: @surface1;
+      --primary-text-color: @text;
+      --secondary-text-color: @subtext0;
+      --shadow-color: fade(@text, 10%);
+      --shadow-light-color: fade(@text, 10%);
+      --danger-color: @red;
+      --important-color: @red;
+      --success-color: @green;
+      --highlights-color: @mauve;
+      --reminder-color: @peach;
+      --note-color: @yellow;
+      --article-color: @peach;
+      --audio-color: @mauve;
+      --document-color: @subtext0;
+      --book-color: @rosewater;
+      --image-color: @green;
+      --video-color: @blue;
+      --broken-color: @maroon;
+      --duplicate-color: @sapphire;
+      --tag-color: @subtext0;
+      --filter-color: @subtext0;
+
+      [data-variant="active"] {
+        background: fade(@accent-color, 10%) !important;
+      }
+      // bookmark edit page header
+      [class^="header-"][data-fancy="true"] {
+        background: linear-gradient(
+          to bottom,
+          var(--background-color) 30%,
+          fade(@base, 60%) 100%
+        );
+      }
+      // bookmark edit page cover selector
+      [class^="cover-"] > [class^="more-"] {
+        background: fade(@accent-color, 85%);
+      }
+      // add bookmark button text color
+      [title="Add Bookmark"] {
+        color: @base;
+      }
+      // disabled add bookmark button
+      [class^="button-"][data-variant="primary"][disabled] {
+        filter: none;
+        opacity: 100%;
+        background: @overlay0;
+        color: @crust;
+      }
+      // dropping bookmark to collection color overlay
+      [class*="isDropping-"] {
+        background-color: fade(@accent-color, 10%);
+      }
+      // sidebar background color
+      .svSidebar {
+        background: var(--sidebar-background-color);
+      }
+      color: @text;
+    }
+    // google login button
+    [href^="https://api.raindrop.io/v1/auth/google"] > span
+    {
+      color: @text;
+    }
+
+    // todo: recolor "Colors circle" icons using custom images (like the chess.com theme)
```

Comment:
Can you expand on this? There might be easier ways to do this (https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#how-do-i-theme-images-and-svgs).

Reply:
![image](https://github.com/user-attachments/assets/3b1a3213-bcc0-4380-9c2e-5c226e7dbeb3)
i had the idea to theme *some* of these default circle color icons.
i don't think it's possible to theme all of them since there are more colors than in the catppuccin color palette
those icons are pngs (e.g. `https://up.raindrop.io/collection/templates/colors/ios1.png`) so i had the idea to make a python script to generate circle icons with catppuccin colors and replace what i can, the way pngs are replaced in the chess.com port.
maybe i can instead replace the circles with generated svgs instead of image
anyway this is just an idea i had and it's low priority and the theme works totally fine without it

Reply:
Cool, I'll resolve this for now!